x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/single_search_after.ts

```diff
@@ -43,11 +47,13 @@ export const singleSearchAfter = async ({
       size: pageSize,
       searchAfterSortId,
     });
+    const start = performance.now();
     const nextSearchAfterResult: SignalSearchResponse = await services.callCluster(
       'search',
       searchAfterQuery
     );
-    return nextSearchAfterResult;
+    const end = performance.now();
+    return { searchResult: nextSearchAfterResult, searchDuration: Number(end - start).toFixed(2) };
```

Comment:
nit -> maybe we can create a small util function to `Number(end - start).toFixed(2)`, you are using it a lot ;)